CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 ### 💡 Enhancements 💡
 
 - Add httpprovider to allow loading config files stored in HTTP (#5810)
+- Add support to propagate trace context of collector's internal spans if `telemetry.allowTraceContextPropagation` featuregate is enabled (#5572)
```

Comment:
Please update.

Reply:
oops! Fixed it - Please review and let me know WDYT?

Reply:
fixed the PR description  as well.

Reply:
thanks @bogdandrutu,  fixed the PR description as well.